drafts/use-cases/6.updating-event.md

```diff
@@ -14,25 +14,25 @@ So I can amend content.
 var client = new HydraClient();
 var event = client.get("/api/events/1")
 event.eventName = "Some new event name.";
-client.invoke(
-    event.getOperations()
-        .filter(op => op[@type].find(type => type === schema.ReplaceAction)[0], event);
+var operation = event.getOperationOfType(schema.ReplaceAction)
```

Comment:
Is schema.ReplaceAction defined somewhere? If not, should we just use the full URL in these examples instead?